data/utils/testDirWith#InName/testTextFile.txt

```diff
@@ -0,0 +1,2 @@
+Test file.
```

Comment:
We should also decide on the naming conventions for test data. Also, as I commented before, it will be nice that the test-resources are located in the `src` to be packaged in the test artifact; otherwise, test won't run if the test jar is isolated.